docs/usage_guide.rst

```diff
@@ -755,17 +770,17 @@ This is solely for importing generic trades. The expected file format is **CSV**
 
 A sample generic trades template can be found below
 
-    .. csv-table:: Rotki Generic Trades Template
+    .. csv-table:: rotki Generic Trades Template
        :file: files/rotki_generic_trades.csv
        :class: longtable
        :header-rows: 1
 
-Rotki Generic Events Import
+rotki Generic Events Import
```

Comment:
In titles and in start of a sentence it looks weird to be lowercase as I told you on discord

```suggestion
Rotki Generic Events Import
```

Reply:
alright then, I'll revert these ones